pkg/config/config.go

```diff
@@ -141,6 +141,17 @@ func (bp Blueprint) ModuleGroupOrDie(mod ModuleID) DeploymentGroup {
 	return g
 }
 
+// GroupIndex returns the index of the input group in the blueprint
+// return -1 if not found
+func (bp *Blueprint) GroupIndex(groupName GroupName) int {
```

Comment:
No need for a reference, just take a value.

Reply:
Done